darts/explainability/shap_explainer.py

```diff
@@ -420,7 +420,11 @@ def summary_plot(
 
         for t in target_components:
             for h in horizons:
-                plt.title("Target: `{}` - Horizon: {}".format(t, "t+" + str(h)))
+                plt.title(
+                    "Target: `{}` - Horizon: {}".format(
+                        t, "t+" + str(h + self.explainers.model.output_chunk_shift)
```

Comment:
I think you can access the model directly like this:

```suggestion
                        t, "t+" + str(h + self.model.output_chunk_shift)
```